xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/DisplayerDocTitle.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <DisplayerLink v-bind="{...$props, ...$attrs, html}"/>
+</template>
+
+<script>
+import DisplayerLink from "./DisplayerLink.vue";
+import displayerMixin from "./displayerMixin.js";
+import displayerLinkMixin from "./displayerLinkMixin.js";
+
+export default {
+  name: "displayer-docTitle",
+  components: {DisplayerLink},
+  mixins: [displayerMixin, displayerLinkMixin],
+  computed: {
+    html() {
+        if (this.entry['doc.title_raw']) {
+        this.logic.footnotes.put('1', 'livedata.footnotes.computedTitle');
```

Comment:
Same as above. The side effect might be unexpected. I guess we can live with this for now.